.gitignore

```
@@ -54,4 +54,6 @@ buck-out/
 
 # Bundle artifact
 *.jsbundle
-.tmp/
\ No newline at end of file
+.tmp/
+
+Pods/
```

shouldn't this be `ios/Pods/`?
```suggestion
ios/Pods/
```

---

I think it works recursively